lib/killbill_client/models/invoice_payment.rb

```diff
@@ -39,6 +41,39 @@ def refund(payment_id, amount, adjustments = nil, user = nil, reason = nil, comm
 
           invoice_payment.refresh(options)
         end
+
+        def create_chargeback(payment_id, amount, currency, effective_date = nil, user = nil, reason = nil, comment = nil, options = {})
```

Comment:
Let's name it `chargeback` for symmetry with the other methods (`refund`, `chargeback_reversal`, etc.).

Reply:
Bump - just wanted to make sure you saw this.

Reply:
Sorry @pierre , I did't saw it. I already pushed the branch with the change.